### ISSUE TITLE
Feature Request: Add option to specify iso filename to download command

### ISSUE THREAD

**Issue by BeatLink** [coreos/coreos-installer#451]
# Feature Request #

## Desired Feature ##
It would be great if there was an option to specify the iso filename while using the download function

**Comment by bgilbert** [comment]
We intentionally preserve the original filename, so that the OS version number is included in the name.  You can capture the stdout of the `download` command to get the name that was chosen.

**Comment by BeatLink** [comment]
Ohh, I see. The reason I asked, was I am writing a small script to download the coreos iso, install an ignition file to it and write the new iso to a usb flash drive. I guess ill have to capture the stdout to get the name. It would be great to at least have the option for those who would want to use it.

**Comment by bgilbert** [comment]
Yeah, makes sense.  The other issue is that with `--format pxe`, we download three separate artifacts, and it's not clear how to cleanly let the user specify all three filenames.
